release/models/macsec/openconfig-macsec.yang

```diff
@@ -627,6 +634,12 @@ revision "2023-06-08" {
          require an ICV";
     }
 
+    leaf include-sci {
```

Comment:
For the Cisco implementation, I assume that this is just not configurable to `false` -- but for the others this switch is needed? If so, LGTM.

Reply:
Yes, I would expect Cisco to accept a "true" value for this and reject a "false" whereas other implementations would accept either value.